3. Installation/4. Manual Installation/Ubuntu/Snaps/README.md

```diff
@@ -18,128 +18,162 @@ sudo apt-get install snapd
 
 
 ## FAQ
+
 If you have questions about snaps best place to ask them is in: [#ubuntu-snap](https://demo.rocket.chat/channel/ubuntu-snap)
 
 ### How do I access my site at a different port?  How do I enable TLS/SSL with my snap?
 
-Check out our guide for enabling caddy: [here](/3.%20Installation/4.%20Manual%20Installation/Ubuntu/Snaps/AutoSSL.md)
+[Check out our guide for enabling caddy](/3.%20Installation/4.%20Manual%20Installation/Ubuntu/Snaps/AutoSSL.md)
 
 ### Ubuntu 16.04 LTS gives message "snap not found", whats wrong?
 
-Make sure you're using x64 or amd64 (or armhf) images, especially on VPS or VMs. x86 is 32 bit and not supported.
+Make sure you're using x64 or amd64 (or armhf) images, especially on VPS or VMs. x86 (32-bit) is not supported.
 
 ### How do I manually update to new release?
 
-Updates happen automatically usually in a 6 hour window from time of release.  If you want it quicker you can do:
+While updates happen automatically usually within 6 hours from time of release, you can update manually by issuing this command:
 ```
 sudo snap refresh rocketchat-server
 ```
 
+### How do I revert to the previous version of Rocket.Chat?
+```
+sudo snap revert rocketchat-server
+```
+
```

Comment:
We need to add a warning here.  If a migration occured on the current version and they roll back it will fail to start.